bin/sozo/src/commands/build.rs

```diff
@@ -80,8 +125,60 @@ mod tests {
             typescript: true,
             unity: true,
             typescript_v2: true,
+            stats: true,
         };
         let result = build_args.run(&config);
         assert!(result.is_ok());
     }
+
+    #[test]
+    fn test_create_stats_table() {
+        // Arrange
+        let contracts_statistics = vec![
+            ContractStatistics {
+                contract_name: "Test1".to_string(),
+                number_felts: 33,
+                file_size: 33,
+            },
+            ContractStatistics {
+                contract_name: "Test2".to_string(),
+                number_felts: 43,
+                file_size: 24,
+            },
+            ContractStatistics {
+                contract_name: "Test3".to_string(),
+                number_felts: 36,
+                file_size: 12,
+            },
+        ];
+
+        let mut expected_table = Table::new();
+        expected_table.set_format(*FORMAT_NO_LINESEP_WITH_TITLE);
+        expected_table.add_row(Row::new(vec![
+            Cell::new_align("Contract", format::Alignment::CENTER),
+            Cell::new_align("Bytecode size (felts)", format::Alignment::CENTER),
+            Cell::new_align("Class size (bytes)", format::Alignment::CENTER),
+        ]));
+        expected_table.add_row(Row::new(vec![
+            Cell::new_align("Test1", format::Alignment::LEFT),
+            Cell::new_align(format!("{}", 33).as_str(), format::Alignment::RIGHT),
+            Cell::new_align(format!("{}", 33).as_str(), format::Alignment::RIGHT),
+        ]));
+        expected_table.add_row(Row::new(vec![
+            Cell::new_align("Test2", format::Alignment::LEFT),
+            Cell::new_align(format!("{}", 43).as_str(), format::Alignment::RIGHT),
+            Cell::new_align(format!("{}", 24).as_str(), format::Alignment::RIGHT),
+        ]));
+        expected_table.add_row(Row::new(vec![
+            Cell::new_align("Test3", format::Alignment::LEFT),
+            Cell::new_align(format!("{}", 36).as_str(), format::Alignment::RIGHT),
+            Cell::new_align(format!("{}", 12).as_str(), format::Alignment::RIGHT),
+        ]));
+
+        // Act
```

Comment:
What `Act` means?

Reply:
Under Act we should place the calls to the functions we are testing.

Following the AAA(Act, Arrange, Assert) test pattern. 
https://automationpanda.com/2020/07/07/arrange-act-assert-a-pattern-for-writing-good-tests/

Sometimes this is used more like a guide.

Reply:
Thank you for the link and the meaning, nice pattern.